tools/webdriver/webdriver/bidi/modules/storage.py

```diff
@@ -1,8 +1,7 @@
-from typing import Any, Dict, Mapping, MutableMapping, Optional, Union
-
+from typing import Any, Dict, Mapping, MutableMapping, Union
```

Comment:
Sorry for working as CI notification again 😬 , but looks like flake8 in unittests job is not happy that `MutableMapping` is imported but not used

Reply:
Thanks for bringing that up!